trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Propulsion/compute_turbofan_geometry.py

```diff
@@ -65,10 +59,13 @@ def compute_turbofan_geometry(turbofan, conditions):
 
     # pack
     turbofan.engine_length    = L_eng_m
-    turbofan.nacelle_diameter = nacelle_diameter
+    turbofan.nacelle_diameter = nacelle_diameter 
+    turbofan.inlet_diameter   = nacelle_diameter/np.sqrt(2.1)
```

Comment:
What is this 2.1 number from?

Reply:
Honestly I don't remember. But, sqrt(2.1) is about 1.5. Which lines up with this source:
https://books.google.com/books?id=NeHoahlhCGMC&pg=PA185&lpg=PA185&dq=nacelle+diameter+to+inlet+diameter&source=bl&ots=DSj5NDl_E7&sig=ACfU3U3i6o9Ak6IOd5MqyvmieiLFKD7wbA&hl=en&sa=X&ved=2ahUKEwiu6My1suTpAhWPlHIEHQ7hB5QQ6AEwAHoECAgQAQ#v=onepage&q=nacelle%20diameter%20to%20inlet%20diameter&f=false